qiskit/providers/ibmq/utils/utils.py

```diff
@@ -56,3 +58,31 @@ def validate_job_tags(job_tags: Optional[List[str]], exception: Type[Exception])
     if job_tags and (not isinstance(job_tags, list) or
                      not all(isinstance(tag, str) for tag in job_tags)):
         raise exception("job_tags needs to be a list or strings.")
+
+
+def setup_logger(logger: Logger) -> None:
+    """Setup the logger for the provider modules with the appropriate level.
+
+    It involves:
+        * Use the `LOG_LEVEL` environment variable to determine the log level
+            for the provider. If it is not set, or is set to an invalid level,
+            the log level is defaulted to `INFO` within `setup_test_logging`.
+        * Use the `STREAM_LOG` environment variable to determine whether the logs
+            should be logged to the console. If it is not set, or set to an invalid
+            level, it defaults to `TRUE`. The valid options are `TRUE` and `FALSE`,
+            case insensitive.
+    """
+    from qiskit.test.utils import setup_test_logging
+
+    log_level = os.getenv('LOG_LEVEL', '')
+    stream_log = os.getenv('STREAM_LOG', '')
```

Comment:
I know they're the env variables used by qiskit test, but they're terrible names. Maybe something more unique like `QISKIT_IBMQ_PROVIDER_LOG_LEVEL`. Also can we have separate options for logging to the screen vs file? These variables should also be documented in a `ibmq/__init__`.

Reply:
I strongly dislike `LOG_LEVEL`, and I don't think having separate control over test vs production logging is a bad thing.